workspace.jsonc

```diff
@@ -293,6 +295,7 @@
         "url-loader": "4.1.1",
         "url-parse": "1.4.7",
         "use-animation-frame": "0.1.0",
+        "uuid": "8.3.2",
```

Comment:
uuid is already installed in package.json, remove it from here or consult with Gilad / David